package.json

```diff
@@ -42,6 +42,7 @@
     "mocha": "^3.2.0",
     "nib": "^1.1.2",
     "pui-react-buttons": "^8.1.0",
+    "pui-react-dropdowns": "^8.3.1",
```

Comment:
Out of interest, why are the Pivotal components being added as dev dependencies, rather than production?